### PR TITLE
on login page show "Login" button, instead of "Submit Query"

### PR DESCRIPTION
**Pull request by Tieske**
Please validate, since I don't know how to test it

### CODE REVIEW

**Comment by hishamhm** [comment]
@leafo is this good to go?

**Comment by leafo** [comment]
yeah it's fine, wasn't aware firefox had different default text. I would probably use `button "Log in"` instead though. (also note "log in" is a verb, "login" is noun)

File: views/user_login.moon [hunk 1]
```diff
@@ -21,7 +21,7 @@ class UserLogin extends require "widgets.page"
         input type: "password", name: "password", id: "password_field"
 
       div class: "button_row", ->
-        input type: "submit"
+        input type: "submit", value: "Login"
```
Comment (hishamhm) [inline views/user_login.moon]:
```suggestion
        input type: "submit", value: "Log in"
```

Comment (Tieske) [reply views/user_login.moon]: thx. I was also looking for a way to do that through the gui 😄

**Pull request merged**